lib/document-to-html.js

```diff
@@ -259,7 +259,8 @@ function DocumentConversion(options, comments) {
     }
     
     function convertTableCell(element, messages, options) {
-        var tagName = options.isTableHeader ? "th" : "td";
+        var isTableHeader = typeof options.isTableHeader !== 'undefined' ? options.isTableHeader : element.isTableHeader;
```

Comment:
I used `typeof options.isTableHeader !== 'undefined'` because the idea was that if `isTableHeader` is set in the options for the table cell, it should override whatever value `element.isTableHeader` ends up being.

Think about this: 
````js
// We want to force this cell to *not* be a table header. 
// It currently *is* a table header
options.isTableHeader = false;
element.isTableHeader = true;

var isTableHeader = options.isTableHeader || element.isTableHeader;
````
The logic for that ends up being:
````js
isTableHeader = false || true;
// isTableHeader = true;
````

Or if we put them the other way around:

````js
isTableHeader = true || false;
//isTableHeader = true;
````